tests/e2e/pageobjects/ide/TopMenu.ts

```diff
@@ -53,7 +53,7 @@ export class TopMenu {
         const submenuItemLocator: By = this.getSubmenuItemLocator(itemText);
 
         // to ensure that drop-down animation is over
-        await this.driverHelper.wait(timeout);
+        await this.driverHelper.wait(5000);
```

Comment:
Would it make sense to extract the magic number into a constant at the top of the file?

Reply:
Hello Tibor, in my eyes the comment is pretty informative, so in my opinion, it doesn't look like a magic number.

Reply:
The variable `timeout` is by default set to exactly 5 seconds [0] so I don't see any value to hardcode it there. Wit the hardcoded value the code is less configurable and you need to send a PR to change this value, instead of setting the env var to change that.

[0] https://github.com/eclipse/che/blob/main/tests/e2e/TimeoutConstants.ts#L158

Reply:
The problem with a variable that if you increase the "TS_SELENIUM_CLICK_ON_VISIBLE_ITEM" variable, you also increase the delay for each time you will invoke this function. I faced that when I increased the "TS_SELENIUM_CLICK_ON_VISIBLE_ITEM" to 60 seconds. The 5 seconds is more than enough for drop-down expanding so I don't see any reason to have there a dynamic value.

Reply:
I have 5 entries in the test with such method, so it means that if I configure "TS_SELENIUM_CLICK_ON_VISIBLE_ITEM=60000" I need to wait 5 minutes (1 minute for each entry) instead 5 seconds in each entry, for drop-down animation quite enough 1-second delay, the 5 seconds is a huge stock.

Reply:
I was thinking about that, working with a drop-down menu is a general case. What do you think if I create a class (for example) DropDownAnimationChecker with the function "waitAnimationEnd". Right now I will add a simple sleep to this method, but when we have time we can rework that for checking and compare container square size (I have already doing a similar method for other repo).

Reply:
Thank you for your explanation. Using "stupid" `sleep` and `wait` without waiting on some event isn't a best practice at all, so I agree with creating some handler as you suggested - e.g. for checking the container size. So, if you can create an issue for this implementation, I'm OK with leaving this hardcoded 5sec wait there for now.

Reply:
@Katka92 Take a look at my last PR https://github.com/eclipse/che/pull/19963/commits/00495856d283ac40bf04bb4e2e75a6e94c8ddacd

Reply:
@Katka92 here is an issue https://github.com/eclipse/che/issues/19965